### ISSUE TITLE
Straight elbow errows end up with untiable knots

### ISSUE THREAD

**Issue by Mrazator** [excalidraw/excalidraw#8736]
Consider the following example, making perfectly straight elbow arrows is not possible, as they often end up with a tiny elbow (knot).

<img height="500px" src="https://github.com/user-attachments/assets/3c4f1031-8973-4773-99a5-c6b6ba54fafe"/>

**Comment by Joyxpro** [comment]
I guess thats why normal arrows are made, use normal arrows for straight ones and elbow arrows for other use cases

**Comment by Mrazator** [comment]
> I guess thats why normal arrows are made, use normal arrows for straight ones and elbow arrows for other use cases

No, it's a bug. Workaround for now is setting slopiness to "architect".
